modules/secured-cluster-services-config.adoc

```diff
@@ -187,11 +187,8 @@ If you do not create this account, you must complete future upgrades manually if
 | Specify `false` to skip the orchestrator secret creation for the Sensor, Collector, and Admission controller.
 
 | `collector.slimMode`
-| Specify `true` if you want to use a slim Collector image for deploying Collector.
-Using slim Collector images with the EBPF collection method requires Central to provide the matching eBPF probe.
-If you are running {product-title} in offline mode, you must download a kernel support package from link:https://install.stackrox.io/collector/support-packages/index.html[stackrox.io] and upload it to Central for slim Collectors to function.
-Otherwise, you must ensure that Central can access the online probe repository hosted at link:https://collector-modules.stackrox.io/[https://collector-modules.stackrox.io/].
-//TODO: Change these links when new links are alive.
+| Deprecated. Specify `true` if you want to use a slim Collector image for deploying Collector.
+There is no longer a difference between the slim and regular Collector images.
 
```

Comment:
I'm thinking we don't need to mention the earlier version thing here and elsewhere in discussions on slim/regular as that may be too much "into the weeds"?